[explorer] Remove targeted builds for explorer (#4631)
MystenLabs/sui: explorer/client/README.md

# How to Switch Environment

> By default, the Sui Explorer will attempt to connect to a locally running RPC server. Refer to [Local RPC Server & JSON-RPC API Quick Start](../../doc/src/build/json-rpc.md) on setting up a Local RPC Server.

Reviewer: ```suggestion
By default, the Sui Explorer attempts to connect to a local RPC server. For more information about using a  local RPC server, see [Local RPC Server & JSON-RPC API Quick Start](../../doc/src/build/json-rpc.md).
```

Reviewer: doesn't it default to Devnet?

Author: This is just for when running the explorer locally. The deployed one defaults to whatever env it's deployed to (e.g explorer.devnet.sui.io points to devnet, explorer.mainnet.sui.io would point to mainnet)

> If you want to use the explorer with another network, you can select your preferred network in the header of the explorer.

Reviewer: Wouldn't they need to choose Custom RPC URL and then enter the IP address to the server?

Author: For custom fullnodes, yes.
